Public/index.php

```diff
@@ -13,4 +14,8 @@
 
 require base_path("config/routes.php");
 
-$router->route($method, $uri);
+try {
+    $router->route($method, $uri);
+} catch (PDOException $e) {
```

Comment:
Handling database exceptions is the database class responsibility. If we keep handling classes exceptions here, we'll end up with handling all application classes exceptions in this file.